extensions/lifecycle/tasks/LifecycleDeleteObjectTask.js

```diff
@@ -0,0 +1,102 @@
+const async = require('async');
+
+const BackbeatTask = require('../../../lib/tasks/BackbeatTask');
+const { attachReqUids } = require('../../../lib/clients/utils');
+
+
+class LifecycleDeleteObjectTask extends BackbeatTask {
+    /**
+     * Process a lifecycle object entry
+     *
+     * @constructor
+     * @param {QueueProcessor} qp - queue processor instance
+     */
+    constructor(qp) {
+        const qpState = qp.getStateVars();
+        super();
+        Object.assign(this, qpState);
+    }
+
+    _checkDate(entry, log, done) {
+        const bucket = entry.getAttribute('target.bucket');
+        const key = entry.getAttribute('target.key');
+        const lastModified = entry.getAttribute('details.lastModified');
+
+        if (lastModified) {
+            const reqParams = {
+                Bucket: bucket,
+                Key: key,
+                IfUnmodifiedSince: lastModified,
```

Comment:
👍

Reply:
That was part of the original code in `LifecycleObjectTask.js`, only renamed the file :wink:

Reply:
ah nice, I actually didn't know about this field and thought it was very useful